open-craft/edx-theme
# Hide enquire button for invitation-only courses, and update Pearson logo

Two changes:
- the logo is replaced: PearsonX → Pearson
- if a course is not listed in the catalog, the _Enquire now_ button isn't shown

**JIRA tickets**: None
**Discussions**: None
**Dependencies**: This is the simple approach to hide the button. We don't need to enable the official `invitation-only` setting (we don't use that because we want to keep the _Enroll now_ button visible).
**Screenshots**: 
old and new logos. And no _Enquire now_ button.
![old](https://user-images.githubusercontent.com/132703/39115567-bd2b0c22-46ea-11e8-982f-d5956d7fa486.png)
![new](https://user-images.githubusercontent.com/132703/39115577-c60b1422-46ea-11e8-922f-6ff2ac8209ca.png)

**Sandbox URL**: https://pearson-test.sandbox.stage.opencraft.hosting/ and https://studio-pearson-test.sandbox.stage.opencraft.hosting/
**Merge deadline**: None

**Testing instructions**:

You can use this course: https://pearson-test.sandbox.stage.opencraft.hosting/courses/course-v1:Testing1+TT11+2018/about

1. check the logo, it shouldn't have X
2. in Studio (URL above), go to a course → advanced settings → and change catalog_visibility from `both` to `about`
3. Verify that the _Enquire now_ is shown when the setting is `both` (and the course is shown in catalog, i.e. https://pearson-test.sandbox.stage.opencraft.hosting/courses/). Keep the URL
4. Verify that the _Enquire now_ is not shown when `about` (and the course isn't shown in catalog, but still has an about page). Note that due to cache, this step can take some minutes to be visible 

**Reviewers**

**Author concerns**: None
**Settings**: None
